src/Cardano/SMASH/Lib.hs

```diff
@@ -6,7 +6,7 @@
 {-# LANGUAGE TypeFamilies        #-}
 {-# LANGUAGE TypeOperators       #-}
 
-module Lib
+module Cardano.SMASH.Lib
```

Comment:
`Lib.hs` is a bit generic I feel. This file contains:

1. servant API types
2. API handler functions
3. utilities for handler functions

It could be split in 2 or 3 modules or be kept as one and named `Api.hs` imo.

Reply:
Split into `API` and `Ŀib`.